### ISSUE TITLE
Infinite loop revalidateCall

### ISSUE THREAD

**Issue by bnason** [Kong/swrv#58]
In version 0.7.0

```
Uncaught (in promise) RangeError: Maximum call stack size exceeded
    at revalidateCall (use-swrv.js?a598:187)
    at revalidateCall (use-swrv.js?a598:187)
    at revalidateCall (use-swrv.js?a598:187)
...
```

https://github.com/Kong/swrv/blob/master/src/use-swrv.ts#L221

`const revalidateCall = async () => revalidateCall();`

should probably be

`const revalidateCall = async () => revalidate();`

**Comment by darrenjennings** [comment]
🤕 fix coming up

**Comment by darrenjennings** [comment]
Released in 0.7.1 and added some missing tests that would have caught this bug.
